Documentation for partitioned dbs (#385)
apache/couchdb-documentation: src/api/partitioned-dbs.rst

.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
.. use this file except in compliance with the License. You may obtain a copy of
.. the License at
..
..   http://www.apache.org/licenses/LICENSE-2.0
..
.. Unless required by applicable law or agreed to in writing, software
.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
.. License for the specific language governing permissions and limitations under
.. the License.

.. _api/partioned-dbs:

=====================
Partitioned Databases
=====================

> Partitioned databases allow for data colocation in a cluster which provides

Reviewer: ```suggestion
Partitioned databases allow for data colocation in a cluster, which provides
```

Reviewer 2: Addressed in 347a6ce.

        Host: localhost:5984

    **Response**:

    .. code-block:: http

        HTTP/1.1 200 OK
        Cache-Control: must-revalidate
        Content-Type: application/json
        Date: Sat, 10 Aug 2013 16:22:56 GMT
        ETag: "1W2DJUZFZSZD9K78UFA3GZWB4"
        Server: CouchDB (Erlang/OTP)
        Transfer-Encoding: chunked

        {
          "offset": 0,
          "rows": [
            {
              "id": "sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf",
              "key": "sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf",
              "value": {
                "rev": "1-05ed6f7abf84250e213fcb847387f6f5"
              }
            }
          ],
          "total_rows": 1
        }

``/db/_partition/partition/_design/design-doc/_view/view-name``
===============================================================

.. http:get:: /{db}/_partition/{partition}/_design/{ddoc}/_view/{view}
    :synopsis: Execute a partitioned query

    :param db: Database name
    :param partition: Partition name
    :param ddoc: Design document id
    :param view: View name

    This endpoint is responsible for executing a partitioned query. The

>     returned view result will only contain rows related to the specified

Reviewer: ```suggestion
    returned view result will only contain rows stored on the specified partition.
```

Is this accurate? If so, it seems more precise.

Author: Re-reading today I went with "only contain rows with the specified partition name". "stored on" seems not quite right and I agree that "related to" seems a bit of a weak association there.

Reviewer 2: Addressed in 347a6ce.
